boards/shields/frdm_stbc_agm01/Kconfig.shield

```diff
@@ -0,0 +1,5 @@
+#
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
The scan code check is failing because you need to add a copyright here

Reply:
Ah, thank you. Not sure why I missed that.